plugins/yivi-web/dom-manipulations.ts

```diff
@@ -237,21 +248,21 @@ module.exports = class DOMManipulations {
   }
 
   _stateEnterPairingCode({ transition, payload }) {
-    const form = this._element.querySelector('.yivi-web-pairing-form');
+    const form = this._element.querySelector('.yivi-web-pairing-form') as HTMLFormElement;
```

Comment:
The `querySelector` method is also generic, but in this case it might be more a matter of preference of how to write it.